lib/src/clojure_lsp/feature/file_management.clj

```diff
@@ -32,7 +32,7 @@
         (shared/client-changes changes db)))))
 
 (defn load-document! [uri text db*]
-  (swap! db* #(assoc-in % [:documents uri] {:v 0 :text text :saved-on-disk false})))
```

Comment:
The pod tests for `format` started failing after defaulting to dep-graph queries. This is the fix—`:documents` includes dep-graph data now, so we can't clobber it.